libs/components/forms/src/lib/modules/file-attachment/file-drop.component.ts

```diff
@@ -55,13 +55,15 @@ export class SkyFileDropComponent implements OnDestroy {
   /**
    * Specifies an ARIA label for the file upload button. This provides a text equivalent for
```

Comment:
```suggestion
   * Specifies an ARIA label for the file upload button. This sets the button's `aria-label` attribute to provide a text equivalent for
```
We don't mention the `aria-label` in the current version of this property description, so the new "For more information" sentence becomes the first reference. I'd tweak the description to add the reference before that new sentence.

Also, FWIW, if it's not too much of a pain, I'd also tweak the similar property descriptions elsewhere to use the same wording. For example, in the flyout component file above where I suggested adding the "For more information" sentence at the end, I'd also change "This sets the flyouts's aria-label attribute to support accessibility." to "This sets the flyouts's aria-label attribute to provide a text equivalent for screen readers to support accessibility." It's a bit longer, but it's more precise.